Dockerfile.template

```diff
@@ -27,6 +27,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
     amd64) ARCH='x64';; \
     ppc64el) ARCH='ppc64le';; \
+    arm64v8) ARCH='arm64';; \
```

Comment:
I don't have arm64 machine. However, here is the [Dockerfile](https://github.com/docker-library/golang/blob/347de3fe79db411821e841f782c156f6fe1d9704/1.9-rc/stretch/Dockerfile#L21) for golang which should work on arm64 machine. The output of `dpkg --print-architecture` should be `arm64`. Sorry for the confusion about the `arm64v8` and `arm64`. Here is my understanding:
- for stackbrew, this is for the docker hub, we need to use `arm64v8`. And the related files in our repo is `architectures` file.
- for Dockerfile, this is for building the image, we need to use `arm64`. Because the output of `dpkg --print-architecture` and `apk --print-arch` is `arm64`

Again I don't have arm64 device, you need to verify it and sorry for the confusion.